explorer/templates/explorer/preview_pane.html

```diff
@@ -78,7 +78,7 @@
                                             </tr>
                                             {% endfor %}
                                         {% else %}
-                                            <tr class="text-center"><td colspan="{{ headers|length }}">Empty Resultset</td></tr>
+                                            <tr class="text-center"><td colspan="{{ headers|length }}">{% blocktrans %}Empty Resultset{% endblocktrans %}</td></tr>
```

Comment:
The choice between  «blocktrans» and «trans»… Actually, may be I little biased toward using «blocktrans» because:
* More agility — to modify to something that contains quotes, tags etc.
* Better readability, because text/message outside tags/quotes

I usially use «trans» if something really simple («one word»), and there are no repetion (like column names).

Reply:
Alright, thanks. It's always good to get someone else's perspective on things 👍